_digest/october-2020/an-effective-guide-to-bullet-journalling.mdx

```diff
@@ -0,0 +1,43 @@
+    <p>Those who are new to bullet journaling struggle to devise how to properly use the technique to organize their schedule and to-do lists in order to reap optimal results of a simultaneous creative outlet and effective organization system. But, knowing the right basics, bullet journaling can help one optimize their time to lead both a productive and healthy life. Today, I will be sharing my guide to bullet journaling 101, and my tips for achieving the above ideal!</p>
+    <p>Before starting to bullet journal, it is crucial for one to find a bullet journal that suits their style and their needs. Not only will it help with creating beautiful bullet journal spreads, but it can also aid with an effective planning system, as you should be comfortable with using the book for long periods of time. The following is a short list of bullet journaling notebooks that I personally find to be most effective (in order of lowest to highest price range).</p>
+    <ol><li><a href="https://www.amazon.com/Dotted-Bullet-Grid-Journal-Notebook/dp/B07FD7FSW3"><strong>Plouma Dotted Grid Notebook</strong></a><strong> ($7.79 CAD)</strong></li></ol>
+    <p>I recommend this book to beginners who are not sure if bullet journaling is the type of planner for them. It is relatively inexpensive and is very effective to draw, colour, and write on.</p>
+    <ol start="2"><li><a href="https://www.amazon.ca/Kesoto-Eco-Friendly-Natural-Hardcover-Notebook/dp/B07D564SLL/ref=pd_lpo_229_t_1/135-8501544-8737535?_encoding=UTF8&amp;pd_rd_i=B07D564SLL&amp;pd_rd_r=64a6e8f6-6322-4632-af0f-f19ddb187225&amp;pd_rd_w=VYUfZ&amp;pd_rd_wg=cdBzC&amp;pf_rd_p=256a14b6-93bc-4bcd-9f68-aea60d2878b9&amp;pf_rd_r=37SPCK43990P40CXBHPB&amp;psc=1&amp;refRID=37SPCK43990P40CXBHPB"><strong>Kesoto Eco-Friendly Cork Hardcover Notebook </strong></a><strong>($16.99 CAD)</strong></li></ol>
+    <p>I recommend this book to people who are semi-dedicated to bullet journaling, or/and are concerned about the environment. It can be used with a wide range of mediums and is also relatively cheap compared to other products on the market that are eco friendly!&nbsp;</p>
+    <ol start="3"><li><a href="https://www.amazon.ca/Leuchtturm1917-Notebooks-Hardcover-Medium-Powder/dp/B07ZXJDY1Y/ref=sr_1_3_sspa?dchild=1&amp;keywords=Leuchtturm1917+Medium+A5+Notebook&amp;qid=1601422379&amp;s=office&amp;sr=1-3-spons&amp;psc=1&amp;spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyMjZCOEZBMVZEMEFFJmVuY3J5cHRlZElkPUEwNzEyMDI5MjRESjNGV0ZTS0FaQyZlbmNyeXB0ZWRBZElkPUEwNDA1NjYxMTNPS0FRVTlOSVZOVCZ3aWRnZXROYW1lPXNwX2F0ZiZhY3Rpb249Y2xpY2tSZWRpcmVjdCZkb05vdExvZ0NsaWNrPXRydWU="><strong>Leuchtturm1917 Medium A5 Notebook </strong></a><strong>($24.95 CAD)</strong></li></ol>
```

Comment:
instead of `<ol start="number"`, change the tags to use 1 `<ol>` for each list (look at the current blog post to see where the list starts/ends) and add paragraph tags after the numbered list item. This is similar to this blog post Bhumika did. https://raw.githubusercontent.com/PuMProfessionals/pump-web-v2/main/_digest/october-2020/incorporate-social-life-in-student-life.mdx
```
<ol> 
  <li>
  <p>
  <li> 
  <p> 
  ... 
</ol> 
```